common/lib/xmodule/xmodule/video_module/video_module.py

```diff
@@ -232,10 +233,26 @@ def get_html(self):
 
         track_url, transcript_language, sorted_languages = self.get_transcripts_for_student()
 
+        # CDN_VIDEO_URLS is only to be used here and will be deleted
+        # TODO(ali@edx.org): Delete this after the CDN experiment has completed.
+        if getattr(settings, 'PERFORMANCE_GRAPHITE_URL', '') != '' and \
+                self.system.user_location == 'CN' and \
+                getattr(settings, 'ENABLE_VIDEO_BEACON', False) and \
+                self.edx_video_id in getattr(settings, 'CDN_VIDEO_URLS', {}).keys():
+            cdn_urls = getattr(settings, 'CDN_VIDEO_URLS', {})[self.edx_video_id]
```

Comment:
I know you promised but the indentationhere throws me off. Just be sure to test this in staging (e.g., non CN users are not sending this data)